integration_tests/steps/kpi.py

```diff
@@ -93,10 +93,26 @@ def available_energy_equals_to_penalties(context, pv_name):
     assert isclose(bills[str(pv.uuid)]["penalty_energy"], penalty_energy, rel_tol=0.0003)
 
 
+def _is_pv(device):
```

Comment:
This is only used by tests though. I understand that we might need this utility function in the future, but IMO functions that are useful only in the test context should remain in the tests and not in the production code.

Reply:
So I shouldn't move as Faizan requested?

Reply:
@lana-shanghai You may keep it separate for now but it would be better to validate pv devices via strategies instances checking like one line below could simply be replaced by `return isinstance(area.strategy, PVStrategy)` since all the other related strategies inherits it as a base strategy for load; Same goes for `_is_load ` and could replace `return isinstance(area.strategy, LoadHoursStrategy)` at L101.